theme/backends.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import unicode_literals
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.backends import ModelBackend
+from django.contrib.auth.tokens import default_token_generator
+from django.db.models import Q
+from django.utils.http import base36_to_int
+from django.core.exceptions import MultipleObjectsReturned
+
+
+User = get_user_model()
+
+
+class CaseInsensitiveMezzanineBackend(ModelBackend):
```

Comment:
In the event of a duplicate email, will this select the first one?

Reply:
no, it'd catch MultipleObjectsReturned and then move on (line 36).  

Checkout this comment to see how I plan on rolling this out.
https://github.com/hydroshare/hydroshare/issues/2781#issuecomment-434430424

Reply:
I don't want to try a password against multiple users, it's a security problem.